rotkehlchen/tests/unit/test_globaldb_asset_updates.py

```diff
@@ -324,7 +331,7 @@ def test_some_updates_are_malformed(assets_updater):
         connection=connection,
         version=999,  # doesn't matter
         text=update_text,
-        conflicts={'ETH': 'remote', 'BTC': 'remote'},
+        conflicts={A_ETH: 'remote', A_BTC: 'remote'},
```

Comment:
Just curious, why change this?

Reply:
the type of conflicts is `asset -> str` and not `str -> str`

Reply:
I just didn't see then, did you change anything in any other place of the code? If not, then how was this working before?

Reply:
because when comparing assets to string we do a string comparasion and we were doing 'ETH' == 'ETH' basically at some point that is what happens when you do `A_ETH == 'ETH'`